samples/client/petstore/java/okhttp-gson-parcelableModel/src/main/java/io/swagger/client/JSON.java

```diff
@@ -113,17 +114,18 @@ public String serialize(Object obj) {
     }
 
     /**
-     * Gson TypeAdapter for JSR310 OffsetDateTime type
+     * Gson TypeAdapter for Joda DateTime type
      */
-    public static class OffsetDateTimeTypeAdapter extends TypeAdapter<OffsetDateTime> {
+    public static class DateTimeTypeAdapter extends TypeAdapter<DateTime> {
 
-        private DateTimeFormatter formatter;
+        private final DateTimeFormatter parseFormatter = ISODateTimeFormat.dateOptionalTimeParser();
+        private final DateTimeFormatter printFormatter = ISODateTimeFormat.dateTime();
 
-        public OffsetDateTimeTypeAdapter() {
-            this(DateTimeFormatter.ISO_OFFSET_DATE_TIME);
+        public DateTimeTypeAdapter() {
+            this(ISODateTimeFormat.dateTime().withOffsetParsed());
         }
 
-        public OffsetDateTimeTypeAdapter(DateTimeFormatter formatter) {
+        public DateTimeTypeAdapter(DateTimeFormatter formatter) {
             this.formatter = formatter;
```

Comment:
This field `formatter` was removed field in favor of `parseFormatter` and `printFormatter` in this PR. This breaks compilation of the generated client. Which field did you intend to set here @Randgalt?